src/NSubstitute/NSubstitute.csproj

```diff
@@ -25,12 +25,11 @@
 		<RepositoryType>git</RepositoryType>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(TargetIsNet5OrNewer)' == 'true'">
-    <DefineConstants>$(DefineConstants);SYSTEM_DIAGNOSTICS_CODEANALYSIS_NULLABILITY</DefineConstants>
+  <PropertyGroup Condition="'$(TargetFramework)' != 'netstandard2.0'">
     <Nullable>enable</Nullable>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(TargetIsNet5OrNewer)' != 'true'">
+  <PropertyGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
     <!-- Nullability does not work nicely for older versions of .NET, so just disable nullability for those versions. -->
     <!-- CS8632 - The annotation for nullable reference types should only be used in code within a '#nullable' annotations context. -->
     <NoWarn>$(NoWarn);CS8632</NoWarn>
```

Comment:
**question:** I'm not across nullability support for different .net versions. How far are we from being able to opt-in to nullability and remove this `<NoWarn />`?

Reply:
Nullability works for .NET Core 3.0+
Until we support .NET Standard or\and .NET Framework we probably need to continue suppress some warning and nullability for older frameworks